options/options.c

```diff
@@ -362,10 +362,10 @@ const struct m_sub_options mp_subtitle_shared_sub_opts = {
         {"sub-visibility", OPT_BOOL(sub_visibility[0])},
         {"secondary-sub-visibility", OPT_BOOL(sub_visibility[1])},
         {"sub-ass-override", OPT_CHOICE(ass_style_override[0],
-            {"no", 0}, {"yes", 1}, {"force", 3}, {"scale", 4}, {"strip", 5}),
+            {"no", 0}, {"yes", 1}, {"scale", 3}, {"force", 4}, {"strip", 5}),
```

Comment:
I don't feel strongly about it so I'll drop this if you do, but I think this is fine because the semantics of yes and scale have changed. Ordering them by how destructive they are to use on ASS subtitles makes sense and this should probably be reflected in documentation and code.

Reply:
I expected the documentation to list the possible values by how destructive they are to use on ASS subtitles. That it currently doesn't was a source of confusion for me when trying to understand what this option does.